README.md

```diff
@@ -59,7 +59,7 @@ The reload flag here will allow the service to restart if you make a change to t
 
 Ajv validator defaults to running on `http://localhost:5002`.
 
-You can override this by setting the `AJV_VALIDATOR_SCHEME`, `AJV_VALIDATOR_HOST`, and `AJV_VALIDATOR_PORT` environment variables. These can be set by creating a `.env` file in the route of the repo file structure. There is a `.development.env` file with defaults that you can look at to see how they need to be set.
+You can override this by setting the `AJV_VALIDATOR_SCHEME`, `AJV_VALIDATOR_HOST`, and `AJV_VALIDATOR_PORT` environment variables. These values are set by the `.development.env` file in the route of the repo file structure. This is linked to `.env` by the `link-development-env` command in the make file, which is triggered automatically on run.
```

Comment:
Just a minor one in the link bit we might want to mention what that link means, that they are synced so that if the env vars are updated in one file it will be updated in the other?